detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>[minor] "To take advantage" -&gt; "To use"</p><p> </p><p>Also, since this <code>process</code> function performs the inferencing, should we define it below in the next section? This section is titled "Load Pretrained Model"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I put <code>process()</code> into a separate section.</p>